integration-tests/rpc-nodes.spec.ts

```diff
@@ -86,6 +86,30 @@ CONFIGS().forEach(
           done();
         });
 
+        it('runView', async (done) => {
+          let chainId: string;
+
+          if (protocol === Protocols.Psithaca2) {
+            chainId = ChainIds.ITHACANET2
+          } else {
+            chainId = ChainIds.HANGZHOUNET
+          }
+
+          const params: RPCRunViewParam = {
+            contract: 'KT1LNMrk8orMQ85zbwK25996dPhDxfSicvKh',
```

Comment:
The contract is deployed on hangzhounet, so the test will fail on ithacanet. I suggest using the knownBigMapContract from the config file which has a getBalance entrypoint too.

Reply:
good catch, will do!